deltacat/compute/converter/utils/convert_task_options.py

```diff
@@ -3,13 +3,14 @@
 from pyiceberg.manifest import DataFile
 from deltacat.compute.converter.model.convert_input_files import ConvertInputFiles
 
-AVERAGE_FILE_PATH_COLUMN_SIZE_BYTES = 80
+AVERAGE_FILE_PATH_COLUMN_SIZE_BYTES = 160
 AVERAGE_POS_COLUMN_SIZE_BYTES = 4
 XXHASH_BYTE_PER_RECORD = 8
 MEMORY_BUFFER_RATE = 2
-# TODO: Add audit info to check this number in practice
 # Worst case 2 as no duplicates exists across all pk
 PYARROW_AGGREGATE_MEMORY_MULTIPLIER = 2
+# Observed base memory usage at the beginning of each process
+BASE_MEMORY_BUFFER = 0.3 * 1024 * 1024 * 1024
```

Comment:
Should we also document that we need at least (300MB + some buffer) of free memory to run the converter? Maybe just the docs for this function should state that the converter requires a node with at least 512MB of free memory?

Reply:
Thanks @pdames, added as `**Memory Requirements:**` doc string in `converter_session()` function. Haven't finalize the naming of al params like maybe `primary key` to `merge key` yet, so doc string is still using the current version of converter. 
Additionally, tested locally that even with just test setup, the process memory is using around 300MB to begin with. Just FYI.